### ISSUE TITLE
Deleting Chat Message/Rooms through API

### ISSUE THREAD

**Issue by uplift** [NodeBB/nodebb-plugin-write-api#141]
I can see in the V2 documentation there is a way to send chat messages but is there a way to delete chat messages (or even rooms)? As far as I can tell currently the only way to delete them is through the Sockets implementation on the Chat page. Is this correct? Could this be added to the API in the future?

**Comment by julianlam** [comment]
@uplift Yes, you are correct, and while additional routes won't be added to the Write API, v3 will likely eventually have a new API for chats.

I'm looking at the implementation now, and my notes don't give me any clue as to why it wasn't migrated along with the other routes, but when I do, I'll likely add in routes for message and room management/deletion as well.
